examples/gallery/ui/pages/table_view_page.slint

```diff
@@ -0,0 +1,44 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+import { HorizontalBox, VerticalBox, StandardTableView, GroupBox} from "std-widgets.slint";
+import { GallerySettings } from "../gallery_settings.slint";
+import { Page } from "page.slint";
+
+export global TableViewPageAdapter := {
+    callback sort_ascending(int);
+    callback sort_descending(int);
+    property <[[StandardListViewItem]]> row_data;
+}
+ 
+export TableViewPage := Page {
+    title: "TableView";
+    description: "StandardListBox can be used to display a list of text elements in columns and rows. It can be imported from \"std-widgets.slint\"";
```

Comment:
```suggestion
    description: "StandardTableView can be used to display a list of text elements in columns and rows. It can be imported from \"std-widgets.slint\"";
```